### PR TITLE
Farkle demo fixes

### LINKED ISSUE #95: Add padding to register button panel

### PR DESCRIPTION
**Pull request by gburgett**
fixes #95

### CODE REVIEW

File: apolloschurchapp/src/util/navigation.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import URL from 'url';
+import querystring from 'querystring';
+
+type Navigator = {
+  navigate(route: string, args?: any): void
+}
+
+export function navigateInApp(rawUrl: string, navigator: Navigator) {
```
Comment (jpowell) [inline apolloschurchapp/src/util/navigation.ts:8]: 👍🏽

File: reports/src/app-links.ts [hunk 2]
```diff
@@ -54,7 +54,7 @@ export class AppLinks {
       { type: 'Tab', title: 'Tracks', link:  'clc://clc/app-link/Tracks' },
       { type: 'Tab', title: 'Connect', link:  'clc://clc/app-link/Connect' },
       { type: 'Tab', title: 'Staff Directory', link:  'clc://clc/app-link/StaffDirectory' },
-      { type: 'Tab', title: 'Settings', link:  'clc://clc/app-link/UserSettings' }
```
Comment (jpowell) [inline reports/src/app-links.ts:57]: do we no longer want the ability to link to settings?

**Review by jpowell — COMMENTED** [review]
do you have updated screenshot for the event registration buttons?

File: reports/src/app-links.ts (hunk 2 above)
Comment (gburgett) [reply reports/src/app-links.ts:57]: This one wasn't working and I never invested time to debug why

**Comment by gburgett** [comment]
image

**Review by jpowell — APPROVED** [review]

**Pull request merged**